www/css/colors.css

```diff
@@ -0,0 +1,9 @@
+:root {
+  --body-text-black: #111;
+  --brand-yellow: #F9D856;
+  --brand-blue: #3881E5;
+  --brand-green: #80BC06;
+  --brand-red: #d02424;
+  --brand-purple: #6b25cf;
+  --grey-outline: 1px solid rgb(204, 204, 204);
```

Comment:
I could call this colors or this could be brand, or it could just be variables.css.  Thoughts?